packages/typespec-vscode/src/telemetry/telemetry-client.ts

```diff
@@ -43,8 +43,14 @@ export class TelemetryClient {
       );
       this._client = undefined;
     } else {
-      // has to convert the TelemetryReporter to any, otherwise it will report error: This expression is not constructable.
-      this._client = new (TelemetryReporter as any)(key);
+      try {
+        this._client = new TelemetryReporter.default(key);
+      } catch (err) {
+        this.logErrorWhenLoggingTelemetry(
+          `Failed to initialize telemetry client with key, error: ${inspect(err)}`,
+        );
+        this._client = undefined;
```

Comment:
is that going to keep trying to created it?

Reply:
No. We only initialize telemetry client once when activate the typespec extension.